krux_marathon_api/cli.py

```diff
@@ -33,9 +33,12 @@ def __init__(self):
         self.marathon_list_apps = self.args.list_apps
         ### Handles files passed via i.e. ~/some-link.json and it will translate
         ### to the proper full location
-        self.marathon_config_file = os.path.realpath(
-            os.path.expanduser(self.args.config_file)
-        )
+        if self.args.config_file:
+            self.marathon_config_file = os.path.realpath(
+                os.path.expanduser(self.args.config_file)
+            )
+        else:
```

Comment:
@thekad, the default value for `--config-file` is `False`.

Also, I would argue it is more readable to still use the `else` here, though I am far from being married to that view.